Readme.md

```diff
@@ -3,57 +3,134 @@
+
+    ```cs
+    public class HomeController : Controller {
+        [HttpGet]
+        public ActionResult Index() {
+            var model = new SpreadsheetData() {
+                DocumentId = Guid.NewGuid().ToString(),
+                DocumentFormat = DocumentFormat.Xlsx,
+                Document = DataHelper.GetDocument()
+            };
+            return View(model);
+        }
+
+        // The method assigned to the Spreadsheet's CallbackRouteAction property in the helper class
+        public ActionResult SpreadsheetPartial() { 
             return SpreadsheetExtension.GetCallbackResult(SpreadsheetSettingsHelper.GetSpreadsheetSettings());
         }
-```
-<br><br><strong>See Also:<br>WebForms Version:</strong><br><a href="https://www.devexpress.com/Support/Center/p/T190812">T190812: ASPxSpreadsheet - How to save and load documents from a database</a>
-### Change Log:
+    }
+    ```
 
+## Files to Review
 
-<strong>16.1:</strong>
-<p>Spreadsheet allows handling certain actions in the Controller now. So, it is possible to save a document to a database using the Spreadsheet ribbon's 'Save' button.</p>
-<strong>15.1:</strong>
+* [SpreadsheetPartial.cshtml](./CS/DXWebApplication23/Views/Home/SpreadsheetPartial.cshtml)
+* [HomeController.cs](./CS/DXWebApplication23/Controllers/HomeController.cs) (VB: [HomeController.vb](./VB/DXWebApplication23/Controllers/HomeController.vb))
+* [DataHelper.cs](./CS/DXWebApplication23/Models/DataHelper.cs) (VB: [DataHelper.vb](./VB/DXWebApplication23/Models/DataHelper.vb))
+* [Index.cshtml](./CS/DXWebApplication23/Views/Home/Index.cshtml)
 
-Now you can use the <a href="https://docs.devexpress.com/AspNet/devexpress.web.mvc.spreadsheetextension.open.overloads">SpreadsheetExtension.Open</a>  method to load a document and call the <a href="https://docs.devexpress.com/AspNet/devexpress.web.mvc.spreadsheetextension.savecopy.overloads">SpreadsheetExtension.SaveCopy</a>  method to save changes.<br><br><strong>For Older Versions:</strong><br>Use <a href="https://documentation.devexpress.com/#CoreLibraries/DevExpressSpreadsheetISpreadsheetComponent_LoadDocumenttopic">ISpreadsheetComponent.LoadDocument</a> to load a document and <a href="https://documentation.devexpress.com/#CoreLibraries/DevExpressSpreadsheetISpreadsheetComponent_SaveDocumenttopic">ISpreadsheetComponent.SaveDocument</a> - to save it.
+## Documentation
 
-<br/>
+* [Creating a Connection String and Working with SQL Server LocalDB](https://learn.microsoft.com/en-us/aspnet/mvc/overview/getting-started/introduction/creating-a-connection-string)
+* [Office Document Management](https://docs.devexpress.com/AspNetMvc/402337/common-features/office-document-management)
 
+## More Examples
 
+* [Spreadsheet for ASP.NET Web Forms - How to save and load documents from a database](https://github.com/DevExpress-Examples/aspxspreadsheet-how-to-save-and-load-documents-from-a-database-t190812)
```

Comment:
This topic has the same potential issue regarding the "save" and "load" generalization under the "from" preposition.